docs/management/deploy-keys.md

```diff
@@ -25,13 +26,19 @@ The commands you can run with a deploy key are limited to the following for secu
 * `releases promote` (only if you specify the release id to promote)
 * `run`
 
+## Custom Roles with Deploy Keys
+
+With the [RBAC](/management/rbac) update, you can assign custom roles to Deploy Keys. This allows you to extend the default set of commands a deploy key can execute, providing more granular control over what operations are permitted. By assigning a custom role, you can tailor a deploy key’s permissions to suit your specific needs.
```

Comment:
Deploy key will only utilize the racks and apps permission, other permission like `users`, `billing` has no affect on it